test/parallel/ddp_optim/test_grad_sync.py

```diff
@@ -192,7 +165,6 @@ def test_ddp(self, overlap_grad_reduce: bool, use_distributed_optimizer: bool):
             m,
             tp_submesh,
             {"parameter": param_sharding_plan, "forward": fwd_sharding_plan},
-            grad_sync={torch.nn.LayerNorm: ["weight", "bias"]},
         )
 
         ddp_m = DDP(
```

Comment:
where is `test/*/devicemesh_api`?

Reply:
incorporated in the second push after randomness.